### PR TITLE
Added a few test cases for minmax.rs

### PR DESCRIPTION
**Pull request by Shreyan11**
As the title says,I've added a few test cases.Please do check if it's all right,or if I need to make any improvements in the test cases,so that I can write unit tests for the rest of the types as well.
I'm open to all suggestions,thank you in advance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<<<<<<< dev
+this is just an explanation of the code and how stuff is connectedme
+=======
```
Comment (stefan-k) [inline README.md:3]: Please remove these lines

File: README.md [hunk 2]
```diff
@@ -141,3 +144,4 @@ at your option.
 ### Contribution
 
 Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any additional terms or conditions.
+>>>>>>> main
```
Comment (stefan-k) [inline README.md:147]: Please remove this line

File: README.md
Comment (stefan-k) [inline README.md:1]: There should be no changes in this file compared to the main branch.

File: argmin-math/src/primitives/l1norm.rs [hunk 3]
```diff
@@ -136,6 +136,7 @@ mod tests {
     }
 
     make_test!(isize);
+
```
Comment (stefan-k) [inline argmin-math/src/primitives/l1norm.rs:139]: Please remove empty line

File: argmin-math/src/primitives/minmax.rs [hunk 4]
```diff
@@ -5,6 +5,7 @@
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
 
+
```
Comment (stefan-k) [inline argmin-math/src/primitives/minmax.rs:8]: Please remove empty line

**Review by stefan-k — CHANGES_REQUESTED** [review]
Thanks for this PR! The tests look good, but I have a few comments regarding other changes.

**Review by Shreyan11 — COMMENTED** [review]
I've made the changed and will make another PR ,please let me know what you think about it.
I'm open to all suggestions

**Pull request closed without merging**